deploy/charts/cert-manager/templates/rbac.yaml

```diff
@@ -504,4 +504,57 @@ subjects:
     namespace: {{ .Release.Namespace | quote }}
     kind: ServiceAccount
 
+---
+
+# Permission to:
+# - Update and sign CertificatSigningeRequests referencing cert-manager.io Issuers and ClusterIssuers
+# - Perform SubjectAccessReviews to test whether users are able to reference Namespaced Issuers
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
It would. Is that really a concern?

Reply:
SubjectAccessReviews are fairly benign for cert-manager to have, and it is only allowed to sign CSRs which explicitly reference cert-manager signers.

Reply:
It's the kind of thing that makes security teams twitchy because it breaks the [principle of least privilege](https://en.wikipedia.org/wiki/Principle_of_least_privilege). Plus, once claimed, it's hard to roll back additional privileges in the future.

I'm not overly concerned by these specific permissions, though - I can just see why a security engineer might be surprised that cert-manager can update CSRs that it never uses.